posthog/management/commands/sync_persons_to_clickhouse.py

```diff
@@ -0,0 +1,151 @@
+
+    if DISTINCT_ID_TABLE in options["tables"]:
+        run_distinct_id_sync(team_id, live_run, deletes)
+
+
+def run_person_sync(team_id: int, live_run: bool, deletes: bool):
+    logger.info("Running person table sync")
+    # lookup what needs to be updated in ClickHouse and send kafka messages for only those
+    persons = Person.objects.filter(team_id=team_id)
+    rows = sync_execute(
+        """
+            SELECT id, max(version) FROM person WHERE team_id = %(team)s GROUP BY id HAVING max(is_deleted) = 0
+        """,
+        {
+            "team": team_id,
+        },
+    )
+    ch_persons_to_version = {row[0]: row[1] for row in rows}
+
+    for person in persons:
+        ch_version = ch_persons_to_version.get(person.uuid, None)
+        if ch_version > person.version:
+            logger.info(
+                f"Clickhouse version ({ch_version}) for '{person.uuid}' is higher than in Postgres ({person.version}). Ignoring."
+            )
+            continue
+        if ch_version != person.version:
+            logger.info(f"Updating {person.uuid} to version {person.version}")
+            if live_run:
+                # Update ClickHouse via Kafka message
+                create_person(
+                    team_id=team_id,
+                    version=person.version,
+                    uuid=str(person.uuid),
+                    properties=person.properties,  # TODO: check that the format is correct
+                    is_identified=person.is_identified,
+                    created_at=person.created_at,  # TODO: check formatting
+                )
+
+    if deletes:
+        logger.info("Processing person deletions")
+        postgres_uuids = {person.uuid for person in persons}
+        for uuid, version in ch_persons_to_version.items():
+            if uuid not in postgres_uuids:
+                logger.info(f"Deleting person with uuid={uuid}")
+                if live_run:
+                    create_person(
+                        uuid=str(uuid),
+                        team_id=team_id,
+                        properties={},
+                        version=int(version or 0)
+                        + 100,  # keep in sync with deletePerson in plugin-server/src/utils/db/db.ts
+                        is_deleted=True,
+                    )
+
+
+def run_distinct_id_sync(team_id: int, live_run: bool, deletes: bool):
+    logger.info("Running person distinct id table sync")
+    # lookup what needs to be updated in ClickHouse and send kafka messages for only those
+    person_distinct_ids = PersonDistinctId.objects.filter(team_id=team_id)
+    rows = sync_execute(
+        """
+            SELECT distinct_id, max(version) FROM person_distinct_id2 WHERE team_id = %(team)s GROUP BY distinct_id HAVING max(is_deleted) = 0
+        """,
+        {
+            "team": team_id,
+        },
+    )
+    ch_distinct_id_to_version = {row[0]: row[1] for row in rows}
+
+    for pdid in person_distinct_ids:
```

Comment:
nit: `did`, `pdid`, `pdi`... can we stick to the standard abbreviation of `pdi` or just use `distinct_id`?